File: foundation/foundation 2 month/lesson4/homework/ex_7.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>
int enter()
{
	return rand()%100;
}

void check(int *ms,int n)
{
	int cnt=0;
	for(int i=0;i<n;i++)
	{
		if(*(ms+i)%2 ==0)
		{
		cnt++;
		}
	}
	printf("\nJuft sonlar: %d ta",cnt);

}

int main()
{
	system("clear");
	srand(time(0));
	int n;
	printf("Elementlar soni: ");
	scanf("%d",&n);
	int arr[n];
	for(int i=0;i<n;i++)
	{
	arr[i]=enter();
	printf("%d,",arr[i]);
	}
	check(arr, n);
return 0;
}

File: full stack/full stack 1 month/lesson3/classwork/ex_7.js
function check(ms)
{
    cnt=0
    a=ms.length
    for(let i=0;i<a;i++)
    {
        if (ms[i]=='0' || ms[i]==1 || ms[i]==2 || ms[i]==3
            || ms[i]==4 || ms[i]==5 || ms[i]==6 || ms[i]==7
            || ms[i]==8 || ms[i]==9 )
        {
            cnt+=1
        }
    }
    if (a>=8 && cnt>0)
    {
        return true
    }
    else
        return false

}

console.log(check("Asliddin2005"))